src/Chainweb/Pact/RestAPI.hs

```diff
@@ -147,6 +149,22 @@ pactLocalWithQueryApi
   . Proxy (PactLocalWithQueryApi v c)
 pactLocalWithQueryApi = Proxy
 
+-- -------------------------------------------------------------------------- --
+-- POST Queries for Pact Poll
+
+type PactPollWithQueryApi_
+    = "poll"
+    :> QueryParam "confirmationDepth" ConfirmationDepth
+    :> ReqBody '[JSON] Poll
+    :> Post '[JSON] PollResponses
+
+type PactPollWithQueryApi v c = PactV1ApiEndpoint v c PactPollWithQueryApi_
```

Comment:
Should we change this in Pact instead of here?

Reply:
I'm following the same approach taken for /local.

I guess there shouldn't even be `PactServer` in `pact` repo? Because that's the scope of the chainweb-node and it's too many efforts to keep the source of truth in the `pact` repo. What's the status of pact dev server `Pact.Server.Server`?

Reply:
no - we need this api to stay static for the `pact -s` use cases. Augmenting it here is more preferable imo.